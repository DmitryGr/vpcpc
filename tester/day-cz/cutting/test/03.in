1281
-310 477
-286 477
-262 477
-238 477
-214 477
-190 477
-166 477
-142 477
-118 477
-94 477
-70 477
-46 477
-22 477
2 477
26 477
50 477
74 477
98 477
122 477
146 477
170 477
194 477
218 477
242 477
266 477
290 477
314 477
338 477
362 477
386 477
410 477
434 477
458 477
482 477
506 477
530 477
578 477
602 477
626 477
650 477
674 477
-310 464
-286 464
-262 464
-238 464
-214 464
-190 464
-166 464
-142 464
-118 464
-94 464
-70 464
-46 464
-22 464
2 464
26 464
50 464
74 464
98 464
122 464
146 464
170 464
194 464
218 464
242 464
266 464
290 464
314 464
338 464
362 464
386 464
410 464
434 464
458 464
482 464
506 464
530 464
554 464
578 464
602 464
626 464
650 464
674 464
-310 451
-286 451
-262 451
-238 451
-214 451
-190 451
-166 451
-142 451
-118 451
-94 451
-70 451
-46 451
-22 451
2 451
26 451
50 451
74 451
98 451
122 451
146 451
170 451
194 451
218 451
242 451
266 451
290 451
314 451
338 451
362 451
386 451
410 451
434 451
458 451
482 451
506 451
530 451
554 451
578 451
602 451
626 451
650 451
674 451
-310 438
-286 438
-262 438
-238 438
-214 438
-190 438
-166 438
-142 438
-118 438
-94 438
-70 438
-46 438
-22 438
2 438
26 438
50 438
74 438
98 438
122 438
146 438
170 438
194 438
218 438
242 438
266 438
290 438
314 438
338 438
362 438
386 438
410 438
434 438
458 438
482 438
506 438
530 438
554 438
578 438
602 438
626 438
650 438
674 438
-310 425
-286 425
-262 425
-238 425
-214 425
-190 425
-166 425
-142 425
-118 425
-94 425
-70 425
-46 425
-22 425
2 425
26 425
50 425
74 425
98 425
122 425
146 425
170 425
194 425
218 425
242 425
266 425
290 425
314 425
338 425
362 425
386 425
410 425
434 425
458 425
482 425
506 425
530 425
554 425
578 425
602 425
626 425
650 425
674 425
-310 412
-286 412
-262 412
-238 412
-214 412
-190 412
-166 412
-142 412
-118 412
-94 412
-70 412
-46 412
-22 412
2 412
26 412
50 412
74 412
98 412
122 412
170 412
194 412
218 412
242 412
266 412
290 412
314 412
338 412
362 412
386 412
410 412
434 412
458 412
482 412
506 412
530 412
554 412
578 412
602 412
626 412
650 412
674 412
-310 399
-286 399
-262 399
-238 399
-214 399
-190 399
-166 399
-142 399
-118 399
-94 399
-70 399
-46 399
-22 399
2 399
26 399
50 399
74 399
98 399
122 399
146 399
170 399
194 399
218 399
242 399
266 399
290 399
314 399
338 399
362 399
386 399
410 399
434 399
458 399
482 399
506 399
530 399
554 399
578 399
602 399
626 399
650 399
674 399
-310 386
-286 386
-262 386
-238 386
-214 386
-190 386
-166 386
-142 386
-118 386
-94 386
-70 386
-46 386
-22 386
2 386
26 386
50 386
74 386
98 386
122 386
146 386
170 386
194 386
218 386
242 386
266 386
290 386
314 386
338 386
362 386
386 386
410 386
458 386
482 386
506 386
530 386
554 386
578 386
602 386
626 386
650 386
674 386
-310 373
-286 373
-262 373
-238 373
-214 373
-190 373
-166 373
-142 373
-118 373
-94 373
-70 373
-46 373
-22 373
2 373
26 373
50 373
74 373
98 373
146 373
170 373
194 373
218 373
242 373
266 373
290 373
314 373
338 373
362 373
386 373
410 373
434 373
458 373
482 373
506 373
530 373
554 373
578 373
602 373
626 373
650 373
674 373
-310 360
-286 360
-262 360
-238 360
-214 360
-190 360
-166 360
-142 360
-118 360
-94 360
-70 360
-46 360
-22 360
2 360
26 360
50 360
74 360
98 360
122 360
146 360
170 360
194 360
218 360
242 360
266 360
290 360
314 360
338 360
362 360
386 360
410 360
434 360
458 360
482 360
506 360
530 360
554 360
578 360
602 360
626 360
650 360
674 360
-310 347
-286 347
-262 347
-238 347
-214 347
-190 347
-166 347
-142 347
-118 347
-94 347
-70 347
-46 347
-22 347
2 347
26 347
50 347
74 347
98 347
122 347
146 347
170 347
194 347
218 347
242 347
266 347
290 347
314 347
338 347
362 347
386 347
410 347
434 347
458 347
482 347
506 347
530 347
554 347
578 347
602 347
626 347
650 347
674 347
-310 334
-286 334
-262 334
-238 334
-214 334
-190 334
-166 334
-142 334
-118 334
-94 334
-70 334
-46 334
-22 334
2 334
26 334
50 334
74 334
98 334
122 334
146 334
170 334
194 334
218 334
242 334
266 334
290 334
314 334
338 334
362 334
386 334
410 334
434 334
458 334
482 334
506 334
530 334
554 334
578 334
602 334
626 334
650 334
674 334
-310 321
-286 321
-262 321
-238 321
-214 321
-190 321
-166 321
-142 321
-118 321
-94 321
-70 321
-46 321
-22 321
2 321
26 321
50 321
74 321
98 321
122 321
146 321
170 321
194 321
218 321
242 321
266 321
290 321
314 321
338 321
362 321
386 321
410 321
434 321
458 321
482 321
506 321
530 321
554 321
578 321
602 321
626 321
650 321
674 321
-310 308
-286 308
-262 308
-238 308
-214 308
-190 308
-142 308
-118 308
-94 308
-70 308
-46 308
-22 308
2 308
26 308
50 308
74 308
98 308
122 308
146 308
170 308
194 308
218 308
242 308
266 308
290 308
314 308
338 308
362 308
386 308
410 308
434 308
458 308
482 308
506 308
530 308
554 308
578 308
602 308
626 308
650 308
674 308
-310 295
-286 295
-262 295
-238 295
-214 295
-190 295
-166 295
-142 295
-118 295
-94 295
-70 295
-46 295
-22 295
2 295
26 295
50 295
74 295
98 295
122 295
146 295
170 295
194 295
218 295
242 295
266 295
290 295
314 295
338 295
362 295
386 295
410 295
434 295
458 295
482 295
506 295
530 295
554 295
578 295
602 295
626 295
650 295
674 295
-310 282
-286 282
-262 282
-238 282
-214 282
-190 282
-166 282
-142 282
-118 282
-94 282
-70 282
-46 282
-22 282
2 282
26 282
50 282
74 282
98 282
122 282
146 282
170 282
194 282
218 282
242 282
266 282
290 282
314 282
338 282
362 282
386 282
410 282
434 282
458 282
482 282
506 282
530 282
554 282
578 282
602 282
626 282
650 282
674 282
-310 269
-286 269
-262 269
-238 269
-214 269
-190 269
-166 269
-142 269
-118 269
-70 269
-46 269
-22 269
2 269
26 269
50 269
74 269
98 269
122 269
146 269
170 269
194 269
218 269
242 269
266 269
290 269
314 269
338 269
362 269
386 269
410 269
434 269
458 269
482 269
506 269
530 269
554 269
578 269
602 269
626 269
650 269
674 269
-310 256
-286 256
-262 256
-238 256
-214 256
-190 256
-166 256
-142 256
-118 256
-94 256
-70 256
-46 256
-22 256
2 256
26 256
50 256
74 256
98 256
122 256
146 256
170 256
194 256
218 256
242 256
266 256
290 256
314 256
338 256
362 256
386 256
410 256
434 256
458 256
482 256
506 256
530 256
554 256
578 256
602 256
626 256
650 256
674 256
-310 243
-286 243
-262 243
-238 243
-214 243
-190 243
-166 243
-142 243
-118 243
-94 243
-70 243
-46 243
-22 243
2 243
26 243
50 243
98 243
122 243
146 243
170 243
194 243
218 243
242 243
266 243
290 243
314 243
338 243
362 243
386 243
410 243
434 243
458 243
482 243
506 243
530 243
554 243
578 243
602 243
626 243
650 243
674 243
-310 230
-286 230
-262 230
-238 230
-214 230
-190 230
-142 230
-118 230
-94 230
-70 230
-46 230
-22 230
2 230
26 230
50 230
74 230
98 230
122 230
146 230
170 230
194 230
218 230
242 230
266 230
290 230
314 230
338 230
362 230
386 230
410 230
434 230
458 230
482 230
506 230
530 230
554 230
578 230
602 230
626 230
650 230
674 230
-310 217
-286 217
-262 217
-238 217
-214 217
-190 217
-166 217
-142 217
-118 217
-94 217
-70 217
-46 217
-22 217
2 217
26 217
50 217
74 217
98 217
122 217
146 217
170 217
194 217
218 217
242 217
266 217
290 217
314 217
362 217
386 217
410 217
434 217
458 217
482 217
506 217
530 217
554 217
578 217
602 217
626 217
650 217
-310 204
-286 204
-262 204
-238 204
-214 204
-190 204
-166 204
-142 204
-118 204
-94 204
-70 204
-46 204
-22 204
2 204
26 204
50 204
74 204
98 204
122 204
146 204
170 204
194 204
218 204
242 204
266 204
290 204
314 204
338 204
362 204
386 204
410 204
434 204
482 204
506 204
530 204
554 204
578 204
602 204
626 204
650 204
674 204
-310 191
-286 191
-262 191
-238 191
-214 191
-190 191
-166 191
-142 191
-118 191
-94 191
-70 191
-46 191
-22 191
2 191
26 191
50 191
74 191
98 191
122 191
146 191
170 191
194 191
218 191
242 191
266 191
290 191
314 191
338 191
362 191
386 191
410 191
434 191
458 191
482 191
506 191
530 191
554 191
578 191
602 191
626 191
650 191
674 191
-310 178
-286 178
-262 178
-238 178
-214 178
-190 178
-166 178
-142 178
-118 178
-94 178
-70 178
-46 178
-22 178
2 178
26 178
50 178
74 178
98 178
122 178
146 178
170 178
194 178
242 178
266 178
290 178
314 178
338 178
362 178
386 178
410 178
434 178
458 178
482 178
506 178
530 178
554 178
578 178
602 178
626 178
650 178
674 178
-310 165
-286 165
-262 165
-238 165
-214 165
-166 165
-142 165
-118 165
-94 165
-70 165
-46 165
-22 165
2 165
26 165
50 165
98 165
122 165
146 165
170 165
194 165
218 165
242 165
266 165
290 165
314 165
338 165
362 165
386 165
410 165
434 165
458 165
482 165
506 165
554 165
578 165
602 165
626 165
650 165
674 165
-310 152
-286 152
-262 152
-238 152
-214 152
-190 152
-166 152
-142 152
-118 152
-94 152
-70 152
-46 152
-22 152
2 152
26 152
50 152
74 152
98 152
122 152
146 152
170 152
194 152
218 152
242 152
266 152
290 152
314 152
338 152
362 152
386 152
410 152
434 152
458 152
482 152
506 152
530 152
554 152
578 152
602 152
626 152
650 152
674 152
-310 139
-286 139
-262 139
-238 139
-214 139
-190 139
-166 139
-142 139
-118 139
-94 139
-70 139
-46 139
-22 139
2 139
26 139
50 139
74 139
98 139
122 139
146 139
170 139
194 139
218 139
242 139
266 139
290 139
314 139
338 139
362 139
386 139
410 139
434 139
458 139
482 139
506 139
530 139
554 139
578 139
602 139
626 139
650 139
674 139
-310 126
-286 126
-262 126
-238 126
-214 126
-190 126
-166 126
-142 126
-118 126
-94 126
-70 126
-46 126
-22 126
26 126
50 126
74 126
98 126
146 126
170 126
194 126
218 126
242 126
266 126
290 126
314 126
338 126
362 126
386 126
410 126
434 126
458 126
482 126
506 126
530 126
554 126
578 126
602 126
626 126
650 126
674 126
-286 113
-262 113
-238 113
-214 113
-190 113
-166 113
-142 113
-118 113
-94 113
-70 113
-46 113
-22 113
2 113
26 113
50 113
74 113
98 113
122 113
170 113
194 113
218 113
242 113
266 113
290 113
314 113
338 113
362 113
386 113
410 113
434 113
458 113
482 113
506 113
530 113
554 113
578 113
602 113
626 113
650 113
674 113
-310 100
-286 100
-262 100
-238 100
-214 100
-190 100
-166 100
-142 100
-118 100
-94 100
-70 100
-46 100
-22 100
2 100
26 100
50 100
74 100
98 100
122 100
146 100
170 100
194 100
218 100
242 100
266 100
290 100
314 100
338 100
362 100
386 100
410 100
434 100
458 100
482 100
506 100
530 100
554 100
578 100
602 100
626 100
650 100
674 100
-310 87
-286 87
-238 87
-214 87
-190 87
-166 87
-142 87
-118 87
-94 87
-70 87
-46 87
-22 87
2 87
26 87
50 87
74 87
98 87
122 87
146 87
170 87
194 87
218 87
242 87
266 87
314 87
338 87
362 87
386 87
410 87
434 87
458 87
482 87
506 87
530 87
554 87
578 87
602 87
626 87
650 87
674 87
