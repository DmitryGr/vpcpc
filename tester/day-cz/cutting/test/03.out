194 0 194 2
-1 295 -3 295
-1 191 -3 191
-1 139 -3 139
-70 0 -70 2
-1 113 -3 113
-190 0 -190 2
0 100 -2 100
-238 0 -238 2
-286 0 -286 2
-214 0 -214 2
-262 0 -262 2
458 0 458 2
-1 451 -3 451
0 308 -2 308
578 0 578 2
-142 0 -142 2
-166 0 -166 2
-118 0 -118 2
-94 0 -94 2
266 0 266 2
314 0 314 2
242 0 242 2
554 0 554 2
0 126 -2 126
674 0 674 2
0 464 -2 464
506 0 506 2
-1 321 -3 321
218 0 218 2
626 0 626 2
-1 347 -3 347
410 0 410 2
650 0 650 2
362 0 362 2
482 0 482 2
386 0 386 2
-22 0 -22 2
602 0 602 2
338 0 338 2
0 360 -2 360
50 0 50 2
434 0 434 2
2 0 2 2
170 0 170 2
-46 0 -46 2
-1 373 -3 373
290 0 290 2
26 0 26 2
0 334 -2 334
-1 399 -3 399
122 0 122 2
530 0 530 2
146 0 146 2
98 0 98 2
-1 477 -3 477
0 230 -2 230
-1 425 -3 425
0 412 -2 412
74 0 74 2
0 386 -2 386
0 256 -2 256
0 438 -2 438
0 152 -2 152
-1 217 -3 217
-1 269 -3 269
0 204 -2 204
-1 165 -3 165
0 282 -2 282
-1 243 -3 243
0 178 -2 178
